src/test/java/com/restassured/exercise/StudentsSkills.java

```diff
@@ -0,0 +1,31 @@
+package com.restassured.exercise;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.cucumber.java.en.Given;
+import  io.restassured.*;
+
+import org.json.simple.JSONObject;
+import org.junit.runner.Request;
+import org.testng.annotations.Test;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+
+
+public class StudentsSkills extends BaseClassAuth{
+	
+	
```

Comment:
this class is empty

Reply:
Changes made locally and will be loaded in the next PR.